Components/Card/card.css

```diff
@@ -0,0 +1,189 @@
+/* Basic Card */
+.basic-card {
+  max-width: 360px;
+  background-color: white;
+  box-shadow: 0px 0px 3px 1px rgba(0, 0, 0, 0.3);
+  font-family: "Poppins", sans-serif;
+}
+
+.basic-card-primary {
+  width: 92%;
+  margin: 0 auto;
+}
+
+.basic-card-primary-h2 {
+  color: black;
+}
+
+.basic-card-primary-h3 {
+  color: #636e72;
+  font-weight: 400;
+  font-size: 1rem;
+  margin-top: -15px;
+}
+
+.basic-card-details {
+  width: 92%;
+  margin: 0 auto;
+}
+
+.basic-card-secondary p {
+  color: #8b9a9b;
+  font-size: 1rem;
+}
+
+.basic-card-actions {
+  width: 92%;
+  margin: 0 auto;
+  padding-bottom: 0.7rem;
+  display: flex;
+  align-items: center;
+}
+
+.basic-card-actions-secondary {
+  margin-left: auto;
+  font-size: 1rem;
+}
+
+.primary-action,
+.secondary-action {
+  height: 2.5rem;
+  background-color: transparent;
+  border: none;
+  display: inline-flex;
+  align-items: center;
+  justify-content: center;
+  cursor: pointer;
+}
+
+.primary-action:hover,
+.secondary-action:hover {
+  background-color: #e6eaeb;
+}
+
+.primary-action {
+  font-weight: bold;
+  color: #4361ee;
+  padding: 0rem 0.6rem;
+  text-transform: uppercase;
+  letter-spacing: 1px;
+  font-family: "Poppins", sans-serif;
+}
+
+.secondary-action {
+  width: 2.5rem;
+  height: 2.5rem;
+  border-radius: 50%;
+  color: #636e72;
+}
+
+/* Text Over Media */
+.basic-card-image {
+  max-width: 100%;
+  object-fit: cover;
+}
+
+.basic-card-image-title {
+  position: relative;
+  display: flex;
+  flex-direction: column;
+}
+
+.text-overlay-h3 {
+  font-weight: 400;
+  font-size: 1rem;
+  margin-top: -20px;
+}
+
+.text-overlay {
+  color: white;
+  position: absolute;
+  bottom: 0;
+  left: 15px;
+}
+
+/* Horizontal Card */
+.horizontal-card {
+  max-width: 400px;
+  background-color: white;
+  box-shadow: 0px 0px 3px 1px rgba(0, 0, 0, 0.3);
+  font-family: "Poppins", sans-serif;
```

Comment:
You are repeating the `font-family` property here and in `basic-card` class. You can give it at the root level instead.